### PR TITLE
Issue #13 - Docker role 

### PR DESCRIPTION
**Pull request by maubarrerag**
Role to install docker

### CODE REVIEW

File: roles/docker/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+---
+# defaults file for docker
```
Comment (donovosoft) [inline roles/docker/defaults/main.yml]: Remove this file, please

File: roles/docker/handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+# handlers file for docker
```
Comment (donovosoft) [inline roles/docker/handlers/main.yml]: Add handlers for stop,start and restart the service

File: roles/docker/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+---
+- include: install.yml
```
Comment (donovosoft) [inline roles/docker/tasks/main.yml]: Add a line or EOF

File: roles/docker/defaults/main.yml (hunk 1 above)
Comment (maubarrerag) [inline roles/docker/defaults/main.yml]: We should keep this file

**Comment by maubarrerag** [comment]
@donovosoft changed

**Pull request merged**